src/main/scala/cromwell/authentication/oauth2/ClientSecrets.scala

```diff
@@ -0,0 +1,8 @@
+package cromwell.authentication.oauth2
+
+trait ClientSecrets {
+  val clientId: String
```

Comment:
hmm, why ? I just want to define an interface that says "whatever class/trait who wants to implement me should have clientId and clientSecret `String`s". Making them `val`s also ensures that they are immutable within that class/trait.